Src/AutoFixture.NUnit3.UnitTest/DummyFixture.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    /// <summary>
+    /// A <see cref="IFixture"/> for the benefit of creating stubs of <see cref="AutoDataAttribute"/> for unit testing
+    /// </summary>
+    public class DummyFixture : IFixture
```

Comment:
AFAICT, this class is specifically used in a single test case, so I'd prefer moving it to the same file as a nested, private class. This keeps it close to where it's used. It also makes it clearer that if the test is ever deleted for some reason, we can also delete the `DummyFixture` class.